src/diracx/cli/__init__.py

```diff
@@ -22,10 +21,9 @@
 @app.async_command()
 async def login(
     vo: str,
-    group: Optional[str] = None,
-    property: Optional[list[str]] = Option(
-        None, help="Override the default(s) with one or more properties"
-    ),
+    group: str | None = None,
```

Comment:
Is the typer problem fixed?

Reply:
Unfortunately not, so I removed the forced changes.